Framework/Core/src/AODReaderHelpers.cxx

```diff
@@ -194,6 +203,9 @@ AlgorithmSpec AODReaderHelpers::rootFileReaderCallback()
       tableMaker(o2::aod::FDDsMetadata{}, AODTypeMask::FDD, "O2fdd");
       tableMaker(o2::aod::UnassignedTracksMetadata{}, AODTypeMask::UnassignedTrack, "O2unassignedtrack");
       tableMaker(o2::aod::Run2V0sMetadata{}, AODTypeMask::Run2V0, "Run2v0");
+      tableMaker(o2::aod::McVtxsMetadata{}, AODTypeMask::McVtx, "O2mcvtx");
+      tableMaker(o2::aod::LabelsMetadata{}, AODTypeMask::Label, "O2labels");
```

Comment:
Looks good to me. The only doubt I have is wether `O2labels` should be `O2label` (or LABEL -> LABELS) and `O2kine` should be `O2mcparticle`.

Reply:
Ok, I will rename the trees in the converter and update this pull request.